src/server/serverInit.ts

```diff
@@ -27,6 +28,7 @@ export const serverInit = () => {
   app.use(bodyParser.json({ limit: '5000kb' }))
 
   resourceCacheControl.init(app)
+  app.use(validateToken)
```

Comment:
Can't this be part of passport middleware when we get the user ?

Reply:
This pr resolves using invalid token (eg mismatch user.id and user.email or missing token.validation which is encrypted)

Reply:
dont we get the user from passport.deserializeUser? we shouldn't store all user in the token . or what do you think?

Reply:
yes, but is this not called only when logging in?
otherwise in the code we use getRequestUser
https://github.com/openforis/fra-platform/blob/39435d6a07c2734ffae5f42bb14bca3e42c6d965/src/server/utils/requests.ts#L48

lets shortly discuss this matter in daily and agree on action points?

Reply:
I think it's called at every request , have you tested ?

Reply:
After testing this I can confirm it is *never* called